packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -324,6 +325,7 @@ const TRANSFORMS = {
         }
       })(),
       expNestedHvm: obj.platform['exp-nested-hvm'] === 'true',
+      ip: extractIpFromVmNetworks(guestMetrics?.networks),
```

Comment:
`mainIpAddress`

Reply:
IMGO, `ip` is simple and it's clear that it's the VM IP.

Reply:
IP is a protocol, this an IP address.

A VM can have multiple IP addresses, I think we should make it clear that it's just one amongst them.